charts/stable/traefik/crds/middlewares.yaml

```diff
@@ -548,6 +548,22 @@ spec:
                       type: string
                     type: array
                 type: object
+              themePark:
+                description: ThemePark middleware contains configuration for the
+                  traefik-themepark plugin. This is a 3rd party plugin that allows for
+                  adding references to `theme.park` (https://theme-park.dev/) themes for
+                  a growing number of appliations.
+                properties:
+                  appName:
+                    description: The name of the supported application to be themed. (ie `sonarr`)
+                    type: string
+                  themeName:
+                    description: The name of the supported theme to be themed. (ie `dark`)
+                    type: string
+                  baseUrl:
+                    description: The base URL to use for loading theme CSS styles from.
+                    type: string
+                type: object
```

Comment:
CRD edits are not going to be allowed.

Reply:
Gotcha, wanted to make sure I had everything covered from a docs perspective.